google/resumable_media/_helpers.py

```diff
@@ -45,6 +47,13 @@ def do_nothing():
     """Simple default callback."""
 
 
+def _base_headers(headers):
+    version = pkg_resources.get_distribution("google-resumable-media").version
+    headers["User-Agent"] = "gcloud-python/{}-resumable-media".format(version)
+    headers["X-Goog-Api-Client"] = "gl-python/" + platform.python_version() + " gccl/{}"
```

Comment:
In python-storage, X-Goog-Api-Client comes out identical to User-Agent which I assumed was the intended design. But that isn't happening in the place in the code I thought it happened, which was storage/_http.py, so I am confused how that came to be.

I think (?) it should be identical to user agent in this case as well.